hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestScmChillMode.java

```diff
@@ -225,36 +210,36 @@ public void testIsScmInChillModeAndForceExit() throws Exception {
 
   }
 
-  @Test(timeout=300_000)
+  @Test(timeout = 300_000)
   public void testSCMChillMode() throws Exception {
-    MiniOzoneCluster.Builder clusterBuilder = MiniOzoneCluster.newBuilder(conf)
-        .setHbInterval(1000)
-        .setNumDatanodes(3)
```

Comment:
Looks like the test was previously using 3 DNs and now it will use just 1 DN. Correct?